components/Search/SearchModal.tsx

```diff
@@ -14,23 +14,9 @@ const SearchModal = ({ search, results, close, onChange, isThreeD }) => {
   })
 
   return (
-    <a.div
-      className="absolute top-0 left-0 bottom-0 right-0 h-screen"
-      css={`
-        z-index: 99;
-      `}
-      style={{ opacity }}
-    >
+    <a.div className="absolute top-0 left-0 bottom-0 right-0 h-screen z-[99]" style={{ opacity }}>
       <button className="opacity-50 bg-gray-900 w-full h-screen" onClick={close}></button>
-      <div
-        className="absolute top-20 left-2/4"
-        css={`
-          max-width: 90%;
-          width: 500px;
-          transform: translateX(-50%);
-          z-index: 100;
-        `}
-      >
+      <div className="absolute top-20 left-2/4 w-[500px] max-w-[90%] z-[100] translate-x-[-50%] transform">
```

Comment:
There is already a `-translate-x-1/2` utility class built into Tailwind that covers `translate-x-[-50%]`.

Also extra `z-index` values like `100` are so frequently used that it might be worth revising the Tailwind config to extend the scale to `100`.

Reply:
Extended the theme and removed the custom class